Makefile

```diff
@@ -1,2 +1,4 @@
 demo:
 	LOCAL=demo go run . -f .env.demo env | sort
+demo-convert:
+	LOCAL=demo2 go run . -f .env.demo env | sort | sed -E 's/(.*)/-e \1/' | tr '\n' ' '
```

Comment:
missing terminal LF : looks like an editor configuration issue ; maybe add a .editorconfig for that ?